### DISCUSSION TITLE
JEI Intergration for Enhancements

### DISCUSSION THREAD

**Discussion by Chrissyofhailfire** [Aizistral-Studios/Enigmatic-Legacy#204] [Ideas]
I've been looking around and at least according to JEI the enhancements Torrent, Wrath, Slayer, sharpshooter Ceaseless, and curse of Nemesis are all from this mod but I'm not able to find a description of what exactly it does in the in-game guide, here in the wiki or on curse forge. having the JEI addition would help those that are trying to understand if they want these enchants on their item. if someone can explain what they do it would help a lot (using better Minecraft plus mod pack)

**Comment by Aizistral** [comment]
Blast, I should change that "get Enchanting Table in your inventory" requirement for unlocking enchantment entries in The Acknowledgment to "interact with Enchanting Table **in any possible way**, you don't even have to craft it, or pick it up, or whatever, just opening the GUI will suffice, I don't care any longer man".

**Comment by zinerith** [comment]
I'm confused as to how the enchantment descriptions are not on the wiki or mentioned anywhere on the mod page. Sure, you can look in the book, but you'd have to be in-game and have said book to do that.
